util/dvsim/Deploy.py

```diff
@@ -203,9 +203,9 @@ def _process_exports(self):
     def _construct_cmd(self):
         """Construct the command that will eventually be launched."""
 
-        args = ["make", "-f", self.flow_makefile, self.target]
+        cmd = "make -f {} {}".format(self.flow_makefile, self.target)
```

Comment:
I don't think this is the right thing to do. If you're struggling to debug things (as described in https://github.com/lowRISC/opentitan/pull/5876#discussion_r606430536) and can't spend the time to fix it properly, maybe it's best to ask someone for help? "I changed this and something I don't understand broke so I'm leaving it be" is not a great way to maintain software, in my opinion.

Reply:
There is some degree of urgency to get this in. Filing an issue so that we fix it later.

Reply:
Filed issue #5945 to ensure this gets dealt with soon.

Reply:
@rswarbrick @sriyerg I agree with Sri - we need to get this in quickly, or we risk getting cutoff from our tools.